uber/site_sections/guest_reports.py

```diff
@@ -96,7 +96,7 @@ def rock_island(self, session, message='', only_empty=None, id=None, **params):
             'only_empty': only_empty
         }
 
-    @site_mappable
+    @site_mappable(download=True)
```

Comment:
This isn't a problem, just curious -- is this because the function has a site_mappable tag (which sets download to False) in addition to the CSV file tag (which sets download to True)?

Reply:
Yeah, pretty sure that's what was happening. I could fix it by flipping the order of the decorators, but I think that'd be a bit obtuse so I just made it explicit.